### PR TITLE
WW-4922: Jackson XML

### PR DESCRIPTION
**Pull request by lukaszlenart**
Implements [WW-4922](https://issues.apache.org/jira/browse/WW-4922)

### CODE REVIEW

**Comment by yasserzamani** [comment]
LGTM :+1: I'm just curious if we can move these to `struts-examples` (because I guess user already can do these himself) ?

**Comment by lukaszlenart** [comment]
Yes, they can but the long plan is to have just a one library to handle JSON/XML serialisation. Also having this in an official package gives users confidence.

**Comment by yasserzamani** [comment]
So wouldn't you like to set it as default now (like what we did with json plugin) ?

File: apps/rest-showcase/src/main/resources/struts.xml [hunk 1]
```diff
@@ -32,6 +32,11 @@
 
     <constant name="struts.convention.package.locators" value="example"/>
 
+<!--
+    <bean name="jacksonXml" type="org.apache.struts2.rest.handler.ContentTypeHandler" class="org.apache.struts2.rest.handler.JacksonXmlHandler" />
+    <constant name="struts.rest.handlerOverride.xml" value="jacksonXml"/>
+-->
+
```
Comment (sdutry) [inline apps/rest-showcase/src/main/resources/struts.xml]: do these commented lines serve any purpose?

Comment (lukaszlenart) [reply apps/rest-showcase/src/main/resources/struts.xml]: An example how to use it but maybe I should move it to struts-examples, wdyt?

Comment (yasserzamani) [reply apps/rest-showcase/src/main/resources/struts.xml]: If so, you'll have to wait for 2.5.16 release.

Comment (lukaszlenart) [reply apps/rest-showcase/src/main/resources/struts.xml]: Yes I know, it's not a problem.

Comment (yasserzamani) [reply apps/rest-showcase/src/main/resources/struts.xml]: As this is a showcase, I think you also have this option to uncomment and let showcase use jackson; a showcase for how override rest content handler.

Comment (lukaszlenart) [reply apps/rest-showcase/src/main/resources/struts.xml]: I added a comment how to use the commented configuration

**Comment by yasserzamani** [comment]
Yes I am. I just was in wait for @sdutry if has something with your question:
> ... maybe I should move it to struts-examples, wdyt?

**Comment by lukaszlenart** [comment]
I see no objections :)

**Comment by yasserzamani** [comment]
Me too :)

**Comment by sdutry** [comment]
I'm sorry for my late response. I didn't mean to be a blocker for the merge.
I've just been busy IRL lately.

**Comment by lukaszlenart** [comment]
@sdutry don't worry, everything is fine :)

**Pull request merged**